modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/file-size-limit/FileSizeMimetypes.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLayout from '@clayui/layout';
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+
+const FileSizeField = ({
+	handleAddClick,
+	handleRemoveClick,
+	index,
+	mimeType,
+	portletNamespace,
+	size,
+}) => {
+	const [mimeTypeValue, setMimeTypeValue] = useState(mimeType);
+	const [sizeValue, setSizeValue] = useState(size);
+
+	return (
+		<ClayLayout.Row className="mt-4">
+			<ClayLayout.Col md="6">
+				<label htmlFor="mimeType">
+					{Liferay.Language.get('mime-type-field-label')}
+
+					<span
+						className="inline-item-after text-muted"
+						title={Liferay.Language.get('mime-type-help-message')}
+					>
+						<ClayIcon symbol="question-circle-full" />
+					</span>
+				</label>
+
+				<ClayInput
+					id="mimeType"
+					name={`${portletNamespace}mimeType_${index}`}
+					onChange={(event) => setMimeTypeValue(event.target.value)}
+					type="text"
+					value={mimeTypeValue}
+				/>
+			</ClayLayout.Col>
+
+			<ClayLayout.Col md="6">
+				<label htmlFor="size">
+					{Liferay.Language.get('maximum-file-size')}
+
+					<span
+						className="inline-item-after text-muted"
+						title={Liferay.Language.get(
+							'maximum-file-size-help-message'
+						)}
+					>
+						<ClayIcon symbol="question-circle-full" />
+					</span>
+				</label>
+
+				<ClayInput
+					id="size"
+					name={`${portletNamespace}size_${index}`}
+					onChange={(event) => setSizeValue(event.target.value)}
+					type="number"
+					value={sizeValue}
+				/>
+
+				{index > 0 && (
+					<ClayButton
+						aria-label={Liferay.Language.get('remove')}
+						className="dm-field-repeatable-delete-button"
+						onClick={() => handleRemoveClick(index)}
+						small
+						title={Liferay.Language.get('remove')}
+						type="button"
+					>
+						<ClayIcon symbol="hr" />
+					</ClayButton>
+				)}
+
+				<ClayButton
+					aria-label={Liferay.Language.get('add')}
```

Comment:
In this case, the `aria-label` and `title` aren't the same but it confuses me, what will happen when I click this button, duplicate or add?

If the `aria-label` and `title` are the same I propose to drop the `aria-label`.

I think `aria-label` is good to add more context like in configs

<img width="1294" alt="image" src="https://user-images.githubusercontent.com/67901/165513831-54312a64-7a48-45e8-903b-06b2bd181e73.png">

Reply:
Ok I see, let me think again about this. To be honest, I think I just copied what they have in DDM, as they have more or less the same component implemented. Good catch!